contracts/ERC1400.sol

```diff
@@ -181,7 +181,7 @@ contract ERC1400 is IERC1400, ERC1410, MinterRole {
     view
     returns (byte, bytes32, bytes32)
   {
-    if(!_checkCertificate(data, 0, 0xf3d490db)) { // 4 first bytes of keccak256(transferByPartition(bytes32,address,uint256,bytes))
+    if(!_checkCertificate(data, 0, bytes4(this.transferByPartition.selector))) { // 0xf3d490db: 4 first bytes of keccak256(transferByPartition(bytes32,address,uint256,bytes))
```

Comment:
Is the `bytes4` cast actually needed? I thought `.selector` returned a `bytes4` already.

Reply:
Yes indeed, I left it there by mistake. Thanks